<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightWidget</class>
 <widget class="QWidget" name="FlightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableFlight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>511</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航司：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCompanyName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlane">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>飞机：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxPlaneName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeatCnt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余座位：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtStartCity">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spxSeatCnt">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartCity">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起点：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEndCity">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>终点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEndCity">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartTime">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间：</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtmStartTime">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEndTime">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>终止时间：</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtmEndTime">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrder">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>订票</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddFlight">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteFlight">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModifyFlight">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
